commit e813506a5b81b0d4ba9dcc6371d2ddb3ec940ed9
Author: Pascal Rapicault (JBoss) <pascal@rapicault.net>
Date:   Fri May 3 12:38:41 2013 -0400

    Bug 407115 - externalize strings

1	0	bundles/org.eclipse.equinox.p2.operations/src/org/eclipse/equinox/internal/p2/operations/Messages.java
1	1	bundles/org.eclipse.equinox.p2.operations/src/org/eclipse/equinox/internal/p2/operations/RequestFlexer.java
1	0	bundles/org.eclipse.equinox.p2.operations/src/org/eclipse/equinox/internal/p2/operations/messages.properties
1	1	bundles/org.eclipse.equinox.p2.operations/src/org/eclipse/equinox/p2/operations/RemediationOperation.java
3	0	bundles/org.eclipse.equinox.p2.ui.sdk/src/org/eclipse/equinox/internal/p2/ui/sdk/ProvSDKMessages.java
2	2	bundles/org.eclipse.equinox.p2.ui.sdk/src/org/eclipse/equinox/internal/p2/ui/sdk/UpdateHandler.java
2	0	bundles/org.eclipse.equinox.p2.ui.sdk/src/org/eclipse/equinox/internal/p2/ui/sdk/messages.properties
3	0	bundles/org.eclipse.equinox.p2.ui/src/org/eclipse/equinox/internal/p2/ui/ProvUIMessages.java
3	4	bundles/org.eclipse.equinox.p2.ui/src/org/eclipse/equinox/internal/p2/ui/actions/UpdateAction.java
1	1	bundles/org.eclipse.equinox.p2.ui/src/org/eclipse/equinox/internal/p2/ui/dialogs/RemediationGroup.java
3	0	bundles/org.eclipse.equinox.p2.ui/src/org/eclipse/equinox/internal/p2/ui/messages.properties
diff --git a/bundles/org.eclipse.equinox.p2.operations/src/org/eclipse/equinox/internal/p2/operations/Messages.java b/bundles/org.eclipse.equinox.p2.operations/src/org/eclipse/equinox/internal/p2/operations/Messages.java
index d71e84e..d340c68 100644
--- a/bundles/org.eclipse.equinox.p2.operations/src/org/eclipse/equinox/internal/p2/operations/Messages.java
+++ b/bundles/org.eclipse.equinox.p2.operations/src/org/eclipse/equinox/internal/p2/operations/Messages.java
@@ -69,6 +69,7 @@ public class Messages extends NLS {
 	public static String RemediationOperation_ProfileChangeRequestProgress;
 	public static String RemediationOperation_ResolveJobName;
 	public static String RemediationOperation_RemediationJobName;
+	public static String RemediationOperation_NoRemedyFound;
 
 	public static String OperationFactory_noAgent;
 	public static String OperationFactory_noIUFound;
diff --git a/bundles/org.eclipse.equinox.p2.operations/src/org/eclipse/equinox/internal/p2/operations/RequestFlexer.java b/bundles/org.eclipse.equinox.p2.operations/src/org/eclipse/equinox/internal/p2/operations/RequestFlexer.java
index c5c2308..3823164 100644
--- a/bundles/org.eclipse.equinox.p2.operations/src/org/eclipse/equinox/internal/p2/operations/RequestFlexer.java
+++ b/bundles/org.eclipse.equinox.p2.operations/src/org/eclipse/equinox/internal/p2/operations/RequestFlexer.java
@@ -223,7 +223,7 @@ public class RequestFlexer {
 	private IProvisioningPlan resolve(IProfileChangeRequest temporaryRequest) {
 		String explainPropertyBackup = null;
 		try {
-			temporaryRequest.setProfileProperty("_internal_user_defined_", "true");
+			temporaryRequest.setProfileProperty("_internal_user_defined_", "true"); //$NON-NLS-1$//$NON-NLS-2$
 			if (provisioningContext != null) {
 				explainPropertyBackup = provisioningContext.getProperty(EXPLANATION_ENABLEMENT);
 				provisioningContext.setProperty(EXPLANATION_ENABLEMENT, Boolean.FALSE.toString());
diff --git a/bundles/org.eclipse.equinox.p2.operations/src/org/eclipse/equinox/internal/p2/operations/messages.properties b/bundles/org.eclipse.equinox.p2.operations/src/org/eclipse/equinox/internal/p2/operations/messages.properties
index 7a01557..4789391 100644
--- a/bundles/org.eclipse.equinox.p2.operations/src/org/eclipse/equinox/internal/p2/operations/messages.properties
+++ b/bundles/org.eclipse.equinox.p2.operations/src/org/eclipse/equinox/internal/p2/operations/messages.properties
@@ -48,6 +48,7 @@ UpdateOperation_UpdateJobName=Updating Software
 RemediationOperation_ProfileChangeRequestProgress=Computing alternate solutions
 RemediationOperation_ResolveJobName=Cannot complete the operation, searching alternate solutions
 RemediationOperation_RemediationJobName=Installing Software
+RemediationOperation_NoRemedyFound=No remedy found
 
 OperationFactory_noAgent=This installation has not been configured properly. No provisioning agent can be found.
 OperationFactory_noIUFound=No IU could be found for {0}.
\ No newline at end of file
diff --git a/bundles/org.eclipse.equinox.p2.operations/src/org/eclipse/equinox/p2/operations/RemediationOperation.java b/bundles/org.eclipse.equinox.p2.operations/src/org/eclipse/equinox/p2/operations/RemediationOperation.java
index 143afc4..ae02400 100644
--- a/bundles/org.eclipse.equinox.p2.operations/src/org/eclipse/equinox/p2/operations/RemediationOperation.java
+++ b/bundles/org.eclipse.equinox.p2.operations/src/org/eclipse/equinox/p2/operations/RemediationOperation.java
@@ -228,6 +228,6 @@ public class RemediationOperation extends ProfileChangeOperation {
 	public IStatus getResolutionResult() {
 		if (currentRemedy != null)
 			return super.getResolutionResult();
-		return remedies.size() > 0 ? Status.OK_STATUS : new Status(IStatus.ERROR, Activator.ID, "No remedy found");
+		return remedies.size() > 0 ? Status.OK_STATUS : new Status(IStatus.ERROR, Activator.ID, Messages.RemediationOperation_NoRemedyFound);
 	}
 }
diff --git a/bundles/org.eclipse.equinox.p2.ui.sdk/src/org/eclipse/equinox/internal/p2/ui/sdk/ProvSDKMessages.java b/bundles/org.eclipse.equinox.p2.ui.sdk/src/org/eclipse/equinox/internal/p2/ui/sdk/ProvSDKMessages.java
index 01d537a..ac7a610 100644
--- a/bundles/org.eclipse.equinox.p2.ui.sdk/src/org/eclipse/equinox/internal/p2/ui/sdk/ProvSDKMessages.java
+++ b/bundles/org.eclipse.equinox.p2.ui.sdk/src/org/eclipse/equinox/internal/p2/ui/sdk/ProvSDKMessages.java
@@ -44,4 +44,7 @@ public class ProvSDKMessages extends NLS {
 	public static String UpdateHandler_NoSitesMessage;
 	public static String UpdateHandler_NoSitesTitle;
 	public static String UpdateHandler_ProgressTaskName;
+	public static String RemediationOperation_ResolveJobName;
+	public static String RemediationOperation_ResolveJobTask;
+
 }
diff --git a/bundles/org.eclipse.equinox.p2.ui.sdk/src/org/eclipse/equinox/internal/p2/ui/sdk/UpdateHandler.java b/bundles/org.eclipse.equinox.p2.ui.sdk/src/org/eclipse/equinox/internal/p2/ui/sdk/UpdateHandler.java
index ddeb4f2..d38b2a8 100644
--- a/bundles/org.eclipse.equinox.p2.ui.sdk/src/org/eclipse/equinox/internal/p2/ui/sdk/UpdateHandler.java
+++ b/bundles/org.eclipse.equinox.p2.ui.sdk/src/org/eclipse/equinox/internal/p2/ui/sdk/UpdateHandler.java
@@ -48,10 +48,10 @@ public class UpdateHandler extends PreloadingRepositoryHandler {
 			} else {
 
 				final RemediationOperation remediationOperation = new RemediationOperation(getProvisioningUI().getSession(), operation.getProfileChangeRequest(), RemedyConfig.getCheckForUpdateRemedyConfigs());
-				ProvisioningJob job2 = new ProvisioningJob("Searching alternate solutions...", getProvisioningUI().getSession()) {
+				ProvisioningJob job2 = new ProvisioningJob(ProvSDKMessages.RemediationOperation_ResolveJobName, getProvisioningUI().getSession()) {
 					@Override
 					public IStatus runModal(IProgressMonitor monitor) {
-						monitor.beginTask("Some items cannot be at the highest version. Searching for the highest common denominator ...", RemedyConfig.getAllRemedyConfigs().length);
+						monitor.beginTask(ProvSDKMessages.RemediationOperation_ResolveJobTask, RemedyConfig.getAllRemedyConfigs().length);
 						return remediationOperation.resolveModal(monitor);
 					}
 				};
diff --git a/bundles/org.eclipse.equinox.p2.ui.sdk/src/org/eclipse/equinox/internal/p2/ui/sdk/messages.properties b/bundles/org.eclipse.equinox.p2.ui.sdk/src/org/eclipse/equinox/internal/p2/ui/sdk/messages.properties
index 19395ce..21e2b3b 100644
--- a/bundles/org.eclipse.equinox.p2.ui.sdk/src/org/eclipse/equinox/internal/p2/ui/sdk/messages.properties
+++ b/bundles/org.eclipse.equinox.p2.ui.sdk/src/org/eclipse/equinox/internal/p2/ui/sdk/messages.properties
@@ -29,3 +29,5 @@ SDKPolicy_PrefPageName=Available Software Sites
 UpdateHandler_NoSitesMessage=There are no update sites to search.  Do you wish to open the "Available Software Sites" preferences?
 UpdateHandler_NoSitesTitle=No Updates Found
 UpdateHandler_ProgressTaskName=Checking for updates...
+RemediationOperation_ResolveJobName=Searching alternate solutions...
+RemediationOperation_ResolveJobTask=Some items cannot be at the highest version. Searching for the highest common denominator ...
\ No newline at end of file
diff --git a/bundles/org.eclipse.equinox.p2.ui/src/org/eclipse/equinox/internal/p2/ui/ProvUIMessages.java b/bundles/org.eclipse.equinox.p2.ui/src/org/eclipse/equinox/internal/p2/ui/ProvUIMessages.java
index 4b74598..69044f1 100644
--- a/bundles/org.eclipse.equinox.p2.ui/src/org/eclipse/equinox/internal/p2/ui/ProvUIMessages.java
+++ b/bundles/org.eclipse.equinox.p2.ui/src/org/eclipse/equinox/internal/p2/ui/ProvUIMessages.java
@@ -205,6 +205,8 @@ public class ProvUIMessages extends NLS {
 	public static String MetadataRepositoryElement_RepositoryLoadError;
 	public static String UpdateAction_UpdatesAvailableMessage;
 	public static String UpdateAction_UpdatesAvailableTitle;
+	public static String UpdateActionRemediationJobName;
+	public static String UpdateActionRemediationJobTask;
 	public static String PlatformUpdateTitle;
 	public static String PlatformRestartMessage;
 	public static String Policy_RequiresUpdateManagerMessage;
@@ -251,6 +253,7 @@ public class ProvUIMessages extends NLS {
 	public static String RemediationPage_BestSolutionBeingInstalledRelaxed;
 	public static String RemediationPage_BestSolutionInstallationRelaxed;
 	public static String RemediationPage_BestSolutionBuilt;
+	public static String RemediationPage_SolutionDetails;
 
 	public static String TrustCertificateDialog_Details;
 	public static String TrustCertificateDialog_Title;
diff --git a/bundles/org.eclipse.equinox.p2.ui/src/org/eclipse/equinox/internal/p2/ui/actions/UpdateAction.java b/bundles/org.eclipse.equinox.p2.ui/src/org/eclipse/equinox/internal/p2/ui/actions/UpdateAction.java
index 3bf6f63..dba0b9a 100644
--- a/bundles/org.eclipse.equinox.p2.ui/src/org/eclipse/equinox/internal/p2/ui/actions/UpdateAction.java
+++ b/bundles/org.eclipse.equinox.p2.ui/src/org/eclipse/equinox/internal/p2/ui/actions/UpdateAction.java
@@ -17,7 +17,6 @@ import java.util.Collection;
 import org.eclipse.core.runtime.*;
 import org.eclipse.core.runtime.jobs.IJobChangeEvent;
 import org.eclipse.core.runtime.jobs.JobChangeAdapter;
-import org.eclipse.equinox.internal.p2.director.ProfileChangeRequest;
 import org.eclipse.equinox.internal.p2.ui.ProvUI;
 import org.eclipse.equinox.internal.p2.ui.ProvUIMessages;
 import org.eclipse.equinox.internal.p2.ui.model.IUElementListRoot;
@@ -79,11 +78,11 @@ public class UpdateAction extends ExistingIUInProfileAction {
 		if (!operation.hasResolved())
 			return Window.CANCEL;
 
-		final RemediationOperation remediationOperation = new RemediationOperation(getSession(), (ProfileChangeRequest) operation.getProfileChangeRequest());
-		ProvisioningJob job = new ProvisioningJob("Searching alternate solutions...", getSession()) {
+		final RemediationOperation remediationOperation = new RemediationOperation(getSession(), operation.getProfileChangeRequest());
+		ProvisioningJob job = new ProvisioningJob(ProvUIMessages.UpdateActionRemediationJobName, getSession()) {
 			@Override
 			public IStatus runModal(IProgressMonitor monitor) {
-				monitor.beginTask("The update operation cannot be completed as requested. Searching alternate solutions ...", RemedyConfig.getAllRemedyConfigs().length);
+				monitor.beginTask(ProvUIMessages.UpdateActionRemediationJobTask, RemedyConfig.getAllRemedyConfigs().length);
 				return remediationOperation.resolveModal(monitor);
 			}
 		};
diff --git a/bundles/org.eclipse.equinox.p2.ui/src/org/eclipse/equinox/internal/p2/ui/dialogs/RemediationGroup.java b/bundles/org.eclipse.equinox.p2.ui/src/org/eclipse/equinox/internal/p2/ui/dialogs/RemediationGroup.java
index 9f65f28..f469644 100644
--- a/bundles/org.eclipse.equinox.p2.ui/src/org/eclipse/equinox/internal/p2/ui/dialogs/RemediationGroup.java
+++ b/bundles/org.eclipse.equinox.p2.ui/src/org/eclipse/equinox/internal/p2/ui/dialogs/RemediationGroup.java
@@ -161,7 +161,7 @@ public class RemediationGroup {
 		resultFoundComposite.setLayout(new GridLayout());
 
 		Group insideFoundComposite = new Group(resultFoundComposite, SWT.NONE);
-		insideFoundComposite.setText("Solution details");
+		insideFoundComposite.setText(ProvUIMessages.RemediationPage_SolutionDetails);
 		GridLayout gridLayout = new GridLayout();
 		gridLayout.marginWidth = 0;
 		gridLayout.marginHeight = 0;
diff --git a/bundles/org.eclipse.equinox.p2.ui/src/org/eclipse/equinox/internal/p2/ui/messages.properties b/bundles/org.eclipse.equinox.p2.ui/src/org/eclipse/equinox/internal/p2/ui/messages.properties
index 0c377d7..349a4ba 100644
--- a/bundles/org.eclipse.equinox.p2.ui/src/org/eclipse/equinox/internal/p2/ui/messages.properties
+++ b/bundles/org.eclipse.equinox.p2.ui/src/org/eclipse/equinox/internal/p2/ui/messages.properties
@@ -128,6 +128,8 @@ AddRepositoryDialog_InvalidURL=The provided location is not valid
 AddRepositoryDialog_LocationLabel=&Location:
 AddRepositoryDialog_NameLabel=&Name:
 
+UpdateActionRemediationJobName=Searching alternate solutions...
+UpdateActionRemediationJobTask=The update operation cannot be completed as requested. Searching alternate solutions ...
 UpdateAction_UpdatesAvailableMessage=Check the updates that you wish to install.
 UpdateAction_UpdatesAvailableTitle=Available Updates
 UpdateOrInstallWizardPage_Size=Size: {0}
@@ -230,6 +232,7 @@ RemediationPage_InstalledSection_AllowInstalledRemoval=Remove items already inst
 RemediationPage_BestSolutionBeingInstalledRelaxed=Keep my installation the same and modify the items being installed to be compatible
 RemediationPage_BestSolutionInstallationRelaxed=Update my installation to be compatible with the items being installed
 RemediationPage_BestSolutionBuilt=Build my own solution:
+RemediationPage_SolutionDetails=Solution Details
 
 IUViewQueryContext_AllAreInstalledDescription=You can uncheck the 'Hide items that are already installed' check box to see all items.
 Label_Profiles=All Software Profiles
